src/server/ban.cpp

```diff
@@ -260,12 +259,12 @@ static lg::log_domain log_server("server");
 
 	void ban_manager::read()
 	{
-		if (filename_.empty() || !filesystem::file_exists(filename_))
+		//if (filename_.empty() || !filesystem::file_exists(filename_))
 			return;
 		LOG_SERVER << "Reading bans from " <<  filename_ << "\n";
 		config cfg;
-		filesystem::scoped_istream ban_file = filesystem::istream_file(filename_);
-		read_gz(cfg, *ban_file);
+		//filesystem::scoped_istream ban_file = filesystem::istream_file(filename_);
```

Comment:
This functionality has been dropped, I think that being able to ban a player is rather important.